src/routes/characters/[slug]/+page.svelte

```diff
@@ -0,0 +1,277 @@
+	import Text from '$lib/components/typography/Text.svelte';
+	import * as Card from '$lib/components/ui/card';
+	import type { Character } from '$lib/types/data/Character';
+	import Badge from '$lib/components/ui/badge/badge.svelte';
+	import * as Tabs from '$lib/components/ui/character-tabs';
+	import * as Table from '$lib/components/ui/table';
+	import { Star } from 'lucide-svelte';
+	import { buttonVariants } from '$lib/components/ui/button';
+	import type { WeaponIndex } from '$lib/types/index/weapon';
+
+	/** @type {import('../../../../.svelte-kit/types/src/routes').PageData} */
+	export let data: {
+		character: CharacterKey;
+		characterData: Character;
+		weaponIndex: WeaponIndex;
+	};
+
+	const splash = S3Service.getCharacterLink(data.character) + '/gacha_splash.webp';
+</script>
+
+<main class="flex-col w-full h-full">
+	<section class="flex w-full gap-4 p-6 box-border h-[75vh]">
+		<Card.Root class="aspect-[1/2] box-border relative">
+			<img
+				class="h-0 min-h-full rounded-md object-cover object-center"
+				src={splash}
+				alt="{data.character} Gacha Splash"
+			/>
+			<div class="flex w-full justify-center absolute bottom-3 left-0">
+				{#each { length: data.characterData.rarity } as _}
+					<Star strokeWidth={0} fill={data.characterData.rarity == 5 ? '#FFB13F' : '#7B5C90'} />
+				{/each}
+			</div>
+		</Card.Root>
+		<!--a href={splash}>{splash}</a-->
+
+		<div class="flex-grow h-full flex gap-2 flex-col box-border">
+			<Text textColor="primary" type="h1">{data.character}</Text>
+			<span class="font-bold">{data.characterData.element} • {data.characterData.weaponType}</span>
+			<div class="flex flex-row w-full">
+				<Badge class="rounded bg-tertiary" variant="default">Sub-DPS</Badge>
+				<!--Find Badge Data Somewhere?-->
+				<!--
+						{#each badges as badge}
+						 <Badge variant="default">{ badge }</Badge>
+						{/each}
+					-->
+			</div>
+			<!--Badges-->
+
+			<Text type="p">{data.characterData.description}</Text>
+
+			<Tabs.Root value="stats" class="w-full flex-grow-0 min-h-0">
+				<Tabs.List class="grid w-full grid-cols-3">
+					<Tabs.Trigger value="stats">Stats</Tabs.Trigger>
+					<Tabs.Trigger value="talents">Talents</Tabs.Trigger>
+					<Tabs.Trigger value="constellations">Constellations</Tabs.Trigger>
+				</Tabs.List>
+				<Tabs.Content value="stats" class="min-h-0 overflow-scroll border border-secondary/50">
+					<Table.Root>
+						<Table.Header>
+							<Table.Row class="p-2 border-b border-secondary/50">
+								{#each data.characterData.ascension[0].stats as stat, i}
+									<Table.Head class="p-2 px-4 font-bold">
+										{stat.label === 'Ascend' ? 'ASC' : stat.label}
+									</Table.Head>
+									{#if i == 0}
+										<Table.Head class="p-2 px-4 font-bold">Level</Table.Head>
+									{/if}
+								{/each}
+							</Table.Row>
+						</Table.Header>
+						<Table.Body>
+							{#each data.characterData.ascension.slice(0,7) as ascension, j}
```

Comment:
I was still working on this but I think I needed the level from the first element but not the data because it just has the per level stat increases because I think originally I displayed both the lowest and highest stats of an ascension phase.